cmd/template/cluster/flag.go

```diff
@@ -15,12 +15,19 @@ import (
 
 const (
 	flagProvider = "provider"
+	flagTplType  = "template-type"
```

Comment:
```suggestion
	flagTemplateType  = "template-type"
```

Flags are not that long so I think its better to align the constants with the full name.

Reply:
Makes sense, done it